code/game/objects/structures/medkit.dm

```diff
@@ -0,0 +1,164 @@
+///Reuses fire extinguisher cabinet wall mount code to create an equivalent for medkits.
```

Comment:
BAD. I haven't double checked but if this is literally just duplicated over from extinguishers you better fix this.